pkgs/applications/misc/golden-cheetah/default.nix

```diff
@@ -16,18 +16,18 @@ let
   };
 in mkDerivation rec {
   pname = "golden-cheetah";
-  version = "3.5";
+  version = "unstable-2021-05-01";
 
   src = fetchFromGitHub {
     owner = "GoldenCheetah";
     repo = "GoldenCheetah";
-    rev = "V${version}";
-    sha256 = "1lyd0b2s3s9c2ppj7l4hf3s4gfzscaaam2pbiaby714bi9nr0ka7";
+    rev = "v3.5-DEV1804-1240-gc9e01353a";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Why are we not using ``3.5-DEV1804-1240-gc9e01353a`` as the version?

Reply:
This has been updated to `3.6-DEV2107`, which is an official tag. I forgot to push parts of this, but did this now.